src/components/stateful-enroll/data/hooks/useStatefulEnroll.js

```diff
@@ -32,9 +34,16 @@ const useStatefulEnroll = ({
   onSuccess,
   onError,
   onBeginRedeem,
+  courseEnrollmentsByStatus,
 }) => {
   const [transaction, setTransaction] = useState();
-
+  const optimizelyHandler = useOptimizelyEnrollmentClickHandler(
+    contentKey,
+    courseEnrollmentsByStatus,
+  );
+  const searchHandler = useTrackSearchConversionClickHandler({
+    eventName: EVENT_NAMES.clickedToEnrollPage,
```

Comment:
Upon closer look, it looks like this event's name (i.e., `edx.ui.enterprise.learner_portal.course.enroll_button.to_enroll.clicked`) indicates it's for clicking the "Enroll" CTA from the course page to bring the learner to enrollment/registration form.

I believe we may want to introduce a net-new Segment event specifically for the enrollment action from within the `handleSuccess` callback for `useStatefulEnroll`, e.g. `edx.ui.enterprise.learner_portal.course.external_enrollment_form.enrolled`. 

That way, we'd have events for:
* Successful enrollment into OCM course.
* Clicking "Enroll" for Exec Ed course from course about page.
* Successfully enrollment into Exec Ed course after successful submission of registration form.